mne/source_estimate.py

```diff
@@ -1342,13 +1337,14 @@ def morph(self, subject_to, grade=5, smooth=None, subjects_dir=None,
         """
         subject_from = _check_subject(self.subject, subject_from)
         if sparse:
-            if grade is not None:
-                raise RuntimeError('grade must be set to None if sparse=True.')
+            from mne.morph import _morph_sparse
             return _morph_sparse(self, subject_from, subject_to, subjects_dir)
         else:
             return morph_data(subject_from, subject_to, self, grade, smooth,
                               subjects_dir, buffer_size, n_jobs, verbose)
```

Comment:
@TommyClausner you have a problem here. You call stc.morph that you deprecated and that inside calls another deprecated function. Why not using the SourceMorph object here?

Reply:
My thought was, that if I make a wrapper anyway I only do it once... - the sparse option was just something I hacked in last minute to make it "complete" (because it's part of the story)

But in general you would have two wrappers instead of one - for me didn't make a difference I suppose